bundles/org.openhab.binding.wundergroundupdatereceiver/src/main/resources/OH-INF/binding/binding.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<binding:binding id="wundergroundupdatereceiver" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:binding="https://openhab.org/schemas/binding/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/binding/v1.0.0 https://openhab.org/schemas/binding-1.0.0.xsd">
+
+	<name>Wunderground Update Receiver Binding</name>
+	<description><![CDATA[
```

Comment:
this text seems to be rendered while choosing the binding when adding a new thing.

- it's rendered incorrect
- it's far to long compared to all the other bindings

![grafik](https://user-images.githubusercontent.com/44850211/108561929-57312e80-72ff-11eb-9f92-95b5c2b594ba.png)

Reply:
Very well. But apparently the documentation about formatting descriptions is incorrect as it states that some html tags including <p> are supported.